Content.Server/Zombies/ZombieSystem.Transform.cs

```diff
@@ -115,51 +117,31 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
             //This is the actual damage of the zombie. We assign the visual appearance
             //and range here because of stuff we'll find out later
             var melee = EnsureComp<MeleeWeaponComponent>(target);
-            melee.ClickAnimation = zombiecomp.AttackAnimation;
-            melee.WideAnimation = zombiecomp.AttackAnimation;
-            melee.Range = 1.2f;
-
-            if (mobState.CurrentState == MobState.Alive)
-            {
-                // Groaning when damaged
-                EnsureComp<EmoteOnDamageComponent>(target);
-                _emoteOnDamage.AddEmote(target, "Scream");
-
-                // Random groaning
-                EnsureComp<AutoEmoteComponent>(target);
-                _autoEmote.AddEmote(target, "ZombieGroan");
-            }
+            melee.ClickAnimation = zombie.Settings.AttackAnimation;
+            melee.WideAnimation = zombie.Settings.AttackAnimation;
+            melee.Range = zombie.Settings.MeleeRange;
+            Dirty(melee);
```

Comment:
1. Range shouldn't be getting set.
2. Need to call Dirty(uid, melee);

Reply:
This is existing functionality, I just exposed the variables to YML. You want me to nerf the zombie range? It currently has a 1.5 multiplier I think.

Reply:
It should just remain as default as there's no good way to telegraph it ingame atm.